evennia/server/evennia_launcher.py

```diff
@@ -732,8 +732,12 @@ def kill(pidfile, signal=SIG, succmsg="", errmsg="",
                   "The PID file 'server/%(pidfile)s' seems stale. "\
                   "Try removing it." % {'pid': pid, 'pidfile': pidfile})
             return
-        print("Evennia:", succmsg)
-        return
+        try:
```

Comment:
Most clean may be to use the win32api call I outlined in issue #1125 to simply turn of the sensitivity to SIGINT in the launcher.